### ISSUE TITLE
Images not appearing (possible wrong path?)

### ISSUE THREAD

**Issue by wildlifehexagon** [literallycanvas/literallycanvas#465]
I tried posting this on Stack Overflow to no avail, but maybe you guys can help me out. I'm trying to implement LiterallyCanvas into a React component. The canvas itself is working, but the images are not being found. At first I thought I just wasn't using the right image URL path, but now I'm not so sure. This embedded file is Canvas.jsx, which is in '/src/containers/'. I am trying to access images that are in '/src/assets/'. To do so, I am using the LiterallyCanvas library's imageURLPrefix option to access '../assets'. Any ideas why the images aren't being found? I'm at a loss here.

```
import React, { Component } from "react";
import Navigation from './Navigation';
const LC = require('literallycanvas');

import styles from '../styles/components/literallycanvas.css';

class Canvas extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      prompts: []
    };
  };

  render () {
    return (
      <div className="container">
        <Navigation />
        <h2>Canvas Component</h2>
        <div>
          <LC.LiterallyCanvasReactComponent imageURLPrefix="../assets" />
          <br />
        </div>
      </div>
    );
  }
}

export default Canvas;
```

**Comment by wildlifehexagon** [comment]
I took another look at this tonight and I'm still having issues getting this to work. I even tried moving the images into an "img" folder in the root directory and accessing them that way, but that didn't help. I'm not sure what's causing this problem since I was able to get the images to load in your npm_webpack_react demo. I would love to use your widget, so any help would be much appreciated.
